Scripts/Services/Expansions/Time Of Legends/Mobiles/DragonTurtle.cs

```diff
@@ -174,7 +174,7 @@ public void DoBubble()
                             if (d == Direction.Up)
                                 d = Direction.North;
                             else
-                                d = d++;
+                                d = ++d;
```

Comment:
Neither the old nor the new lines make sense here, left-hand assignment isn't necessary.